pkg/sidecar/link.go

```diff
@@ -96,48 +96,67 @@ func (l *NetlinkLink) init(idx uint16) error {
 }
 
 // Sets the bandwidth (in bytes per second).
-func (l *NetlinkLink) setBandwidth(idx uint16, rate uint64) error {
+func (l *NetlinkLink) setHtb(idx uint16, attrs netlink.HtbClassAttrs) error {
```

Comment:
What does `Htb` stand for?

Reply:
The htb queuing class. https://linux.die.net/man/8/tc-htb

Reply:
Ah, I forgot to update the docs. Thanks for catching that.